### DISCUSSION TITLE
git插件问题

### DISCUSSION THREAD

**Discussion by rose8928** [opensumi/core#1998] [Q&A]
打扰，我安装了git,git-base,git-ui但是首页clone repository是灰色，并且点击无反应。我的customVSCodeEngineVersion版本是1.60.2.
git版本是 {
"name": "git",
"version": "1.62.3"
},
{
"name": "git-ui",
"version": "1.55.2"
},
{
"name": "git-base",
"version": "1.64.0-next.68976705968"
},
<img width="662" alt="git-fail" src="https://user-images.githubusercontent.com/35814595/203239490-2c4d54e3-9542-45b4-8c62-681c1649dce0.png">

但是git模块打开是ok的
<img width="600" alt="git-ok" src="https://user-images.githubusercontent.com/35814595/203239516-4e9eda35-0d1f-445b-b8da-5d06e462727f.png">

我以为版本不对，我或者修改了customVSCodeEngineVersion，或者适配相应的git，git-base版本，都无效，而且有些版本报错。
是为什么呢？git这个模块是可用的吗？
因为我看到我clone的opensumi-electron的customVSCodeEngineVersion版本号就是1.60.2，且build/extension.json里的的git,git-base,git-ui和我用的都是一样的，
<img width="632" alt="git-opensumi-ok" src="https://user-images.githubusercontent.com/35814595/203240271-951248ab-833e-4e0e-b7d7-9edb71d5d60e.png">
这个也是灰色的，但是点击能打开quickopen，如图：

**Comment by erha19** [comment]
@rose8928 目前 welcome 页面上面的克隆按钮确实是不可用的，这块可以记个 issue 我们处理一下，对于 `clone repository` 是灰色的问题，不确定是否与你本地修改有关，我这边用 https://github.com/opensumi/ide-electron 项目运行是正常的。

**Reply by rose8928** [reply]
你的意思是clone repository点击没有反应是正常的吗？但是我用的ide-electron项目运行，点击是可以打开quick-open的呀。

**Reply by erha19** [reply]
行为正常，但功能不正常，我没细看，猜测是 Git 插件实现该部分功能依赖了某些我们不支持的 VS Code 内置命令

**Reply by rose8928** [reply]
了解了
